jdaviz/configs/cubeviz/plugins/moment_maps/moment_maps.py

```diff
@@ -145,15 +137,11 @@ def vue_calculate_moment(self, *args):
             self.app.add_data_to_viewer(self.viewer_to_id.get(self.selected_viewer),
                                         label, clear_other_data=True)
 
-    def vue_save_as_fits(self, event):
-        self.moment.write(self._filename)
-        # Let the user know where we saved the file (don't need path if user
-        # specified a full filepath
-        if re.search("/", self._filename) is None:
-            wd = pathlib.Path.cwd()
-            full_path = wd / pathlib.Path(self._filename)
-        else:
-            full_path = self._filename
-        msg = SnackbarMessage("Moment map saved to {}".format(str(full_path)),
-                              sender=self, color="success")
-        self.hub.broadcast(msg)
+    def vue_save_as_fits(self, *args):
+        if self.moment is None or not self.filename:  # pragma: no cover
+            return
+
+        self.moment.write(self.filename)
+        # Let the user know where we saved the file.
+        self.hub.broadcast(SnackbarMessage(
+            f"Moment map saved to {os.path.abspath(self.filename)}", sender=self, color="success"))
```

Comment:
I don't know why there was a need for a complicated logic just for a snackbar message. I think it is always good to report back the full path so there is no doubt there the output file is.